ChildProject/annotations.py

```diff
@@ -20,6 +20,20 @@ class AnnotationManager:
         IndexColumn(name = 'format', description = 'input annotation format', regex = r"(TextGrid|eaf|rttm)", required = True)
     ]
 
+    SEGMENTS_COLUMNS = [
+        IndexColumn(name = 'annotation_file', description = 'raw annotation path relative to /raw_annotations/', required = True),
+        IndexColumn(name = 'segment_onset', description = 'segment start time in seconds', regex = r"[0-9]{1,}(\.[0-9]{3})?", required = True),
+        IndexColumn(name = 'segment_offset', description = 'segment end time in seconds', regex = r"[0-9]{1,}(\.[0-9]{3})?", required = True),
+        IndexColumn(name = 'speaker_id', description = '', required = True),
```

Comment:
Can you please help me fill empty descriptions here ? :) @alecristia

Reply:
done!

Reply:
done)